packages/ifpack2/src/Ifpack2_Details_OverlappingRowGraph_def.hpp

```diff
@@ -88,12 +88,14 @@ OverlappingRowGraph<GraphType>::getComm () const
 }
   
 
+#ifdef TPETRA_ENABLE_DEPRECATED_CODE
 template<class GraphType>
 Teuchos::RCP<typename GraphType::node_type> 
 OverlappingRowGraph<GraphType>::getNode () const
 {
   return nonoverlappingGraph_->getNode();
```

Comment:
This calls a deprecated method, so it will emit a redundant deprecated warning.  Does any caller depend on the returned `RCP<Node>` being nonnull?  My guess is no, so we could just return `Teuchos::null` here and avoid the extra deprecated warning.  Ditto for other calls to `getNode` in this PR.

Reply:
Done